### ISSUE TITLE
False positive error for case @covers Class::<!private> in 2020.1 EAP

### ISSUE THREAD

**Issue by MaXal** [kalessil/phpinspectionsea#1505]
Due to the parsing changes of `ClassToCover::<!public>` and similar in 2020.1 EAP the following code produces Error in `PHP | Php Inspections (EA Extended) | PhpUnit | PhpUnit: bugs and best practices` inspection:
```
<?php
class FooMe{
}
class MyTest extends TestClass{
    /**
     * @covers FooMe::<!public>
     */
    public function testMe()
    {
    }
}
```

The error is: `@covers referencing to a non-existing entity 'Foo::<!public>'`

Reference link: https://youtrack.jetbrains.com/issue/WI-51108

**Comment by kalessil** [comment]
Oh, thank you for reporting @MaXal, I'll take care of fixing this.

**Comment by kalessil** [comment]
Fixed!
